.github/workflows/pyinstaller_gui_mac.yaml

```diff
@@ -19,9 +19,9 @@ jobs:
 #          ref: master
       - uses: actions/setup-python@v4
         with:
-          python-version: "3.10"
+          python-version: "3.11"
       - name: install tkinter
-        run: brew install python-tk@3.10
+        run: brew install python-tk@3.11
```

Comment:
I think this might not be needed because tkinter a native python library and am happy to remove it. It also doesn't hurt to have because it currently works and the GitHub workflow minutes for our repository is free since we are open-source code